design-reviews/recipes/milestone-epic-design-review-recipe.md

```diff
@@ -0,0 +1,79 @@
+# Milestone / Epic Design Review Recipe
+
+## Things to keep in mind
+
+* As with all other aspects of the project, design reviews must provide a friendly and safe environment so that any team member feels comfortable proposing a design for review and can use the opportunity to grow and learn from the constructive / non-judgemental feedback from peers and subject matter experts (see [Team Agreements](../../team-agreements)).
+* Design reviews should be lightweight and should not feel like an additional process overhead.
+* Tech Lead can usually provide guidance on whether a given epic/milestone needs a design review and can help other team members in preparation.
+* This is *not* a strict template that must be followed and teams should not be bogged down with polished "design presentations".
+* Think of the recipe below as a "menu of options" for potential questions to think through in designing this epic. Not all sections are required for every epic. Focus on sections and questions that are most relevant for making the decision and rationalizing the trade offs.
+* Milestone/epic design is considered high-level design but is usually more detailed than the design included in the Game Plan, but will likely re-use some of the technologies, non-functional requirements, and constraints mentioned in the Game Plane.
+* As the team learned more about the project and further refined the scope of the epic, they may specifically call out notable changes to the overall approach and, in particular, highlight any unique deployment, security, private, scalability, etc. characteristics of this milestone.
+
+***
+
+# [Design Title]
+
+* Milestone / Epic: [Name](http://link-to-work-item)
+* Project / Engagement: [Project Engagement]
+* Authors: [Authors]
+
+## Overview / Problem Statement
+
+> Describe the milestone/epic with a high-level summary and a problem statement. Consider including or linking to any additional background (e.g. Game Plan or Checkpoint docs) if it is useful for historical context.
+
+## Goals / In-Scope

```

Comment:
Would Goals/Non-Goals be set in the work item that represents the epic in whatever ALM tool being used (AzDO, Jira, Github, etc)? If so why, should we repeat this here?

Reply:
Agree that Goals/Non-Goals of the epic would be in the work item corresponding to this epic. I was thinking these sections could be useful in the design review recipe for a few bullet points describing that's specifically in scope or out of scope for the design review in question. @wbreza since these are from the original template, what do you think?

Reply:
@charleszipp @arsenvlad As with most things, I think it is context sensitive. If I want to avoid going down a rabbit hole or people worrying about an out of scope (non-goal) of the review, I will mention it so that we can move past the point.

Reply:
Thanks Ian!